go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/companieshouse/chs-delta-api
 
-go 1.12
+go 1.15
```

Comment:
Are we sure we want to use go 1.15?? This will need creation of new. tasks in concourse and probably we need to let platform know?

Reply:
For transparency. We will be going with go v1.12 and on the side we will get support and begin implementing latest versions in concourse etc (v1.15+...).